source/server/overload_manager_impl.cc

```diff
@@ -194,8 +194,8 @@ class ThreadLocalOverloadStateImpl : public ThreadLocalOverloadState {
                            int64_t increment) override {
     const auto proactive_resource = proactive_resources_->find(resource_name);
     if (proactive_resource == proactive_resources_->end()) {
-      ENVOY_LOG_MISC(warn, " {Failed to allocate unknown proactive resource }");
-      // Resource monitor is not configured.
+      ENVOY_LOG_MISC(warn,
+                     " {Failed to allocate resource usage, resource monitor is not configured}");
```

Comment:
Any reason why the log reason needs to be in braces? vs without braces? (here and other logging in this file)